src/tests/multihost/basic/conftest.py

```diff
@@ -208,6 +211,14 @@ def set_case_sensitive_false(session_multihost):
                                                    '/etc/sssd/sssd.conf')
     session_multihost.master[0].service_sssd('restart')
 
+    def restore_sssd():
+        """ Restore sssd.conf """
+        restore_sssd = 'cp -f /etc/sssd/sssd.conf.orig /etc/sssd/sssd.conf'
+        session_multihost.master[0].run_command(restore_sssd)
+        session_multihost.master[0].service_sssd('restart')
+        time.sleep(5)
```

Comment:
Why do we need the sleep here? Doesn't systemctl block until the service starts?